libbeat/beat/beat.go

```diff
@@ -469,8 +469,30 @@ func (b *Beat) registerTemplateLoading() error {
 		}
 	}
 
-	esConfig := b.Config.Output["elasticsearch"]
+	// Check if outputting to file is enabled, and output to file if it is
```

Comment:
This comment confused me at first because the check for the file only happens later, but I see you meant it for the "complete" block